index.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
```

Comment:
- [ ] Please use polite suggestions like please "I think it would be better to use semantic tags" Please put your div tag in a section tag you don't want to sound like you are threatening.

- [ ] Please always includes an explanation, a "why", for each requested change, the programmer always want an explanation they want to know why you are requesting the changes
This is a well-explained request: *Why don't you use an article tag for the .first-article element instead of a div, that would improve the HTML structure and make the page more accessible*
![image](https://user-images.githubusercontent.com/74744787/159242238-c150a08b-fb1c-49b1-a16a-4b27e52ed1d7.png)